src/main/java/com.microsoft.azure/NetworkAclSample.java

```diff
@@ -1,34 +1,34 @@
 package com.microsoft.azure;

```

Comment:
It is curious that the path of the file is `src/main/java/com.microsoft.azure/`. Usually it should be `src/main/java/com/azure/resourcemanager`.

No a blocking issue, this still serve the purpose of a sample.

Reply:
Changed the path of the file to `src\main\java\com\microsoft\azure`.